Lib9c/Action/TransferAsset.cs

```diff
@@ -117,8 +118,56 @@ public override IAccount Execute(IActionContext context)
             }
 
             TransferAsset3.CheckCrystalSender(currency, context.BlockIndex, Sender);
+            if (state.TryGetState(Recipient, out Dictionary dictionary))
```

Comment:
Can you consider MonsterCollection state?😅

Reply:
I fixed it!